src/cart/src/cart/crt_register.c

```diff
@@ -469,10 +469,10 @@ crt_proto_register_common(struct crt_proto_format *cpf)
 
 	if (cpf->cpf_ver > CRT_PROTO_MAX_VER) {
 		D_ERROR("Invalid version number %d, max version number is "
-			"%lu.\n", cpf->cpf_ver, CRT_PROTO_MAX_VER);
+  		"%lu.\n", cpf->cpf_ver, CRT_PROTO_MAX_VER);
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs